src/components/seed-search-form.jsx

```diff
@@ -189,15 +192,21 @@ class SeedSearchForm extends React.Component {
     )
   }
 
+  seedGenre(event) {
+    const seedInput = document.getElementById("seed")
```

Comment:
Usually it's a red flag if you're using functions like `getElementById` or `querySelector` in a React app.